app/src/main/kotlin/com/simplemobiletools/keyboard/views/MyKeyboardView.kt

```diff
@@ -1632,18 +1639,32 @@ class MyKeyboardView @JvmOverloads constructor(context: Context, attrs: Attribut
             } else {
                 mToolbarHolder?.autofill_suggestions_holder?.addView(it)
             }
-
-            // make room on suggestion toolbar for inline views
-            mToolbarHolder?.suggestions_items_holder?.gravity = Gravity.NO_GRAVITY
-            mToolbarHolder?.clipboard_value?.maxWidth = resources.getDimensionPixelSize(R.dimen.suggestion_max_width)
+            updateSuggestionsToolbarLayout()
         }
     }
 
     @RequiresApi(Build.VERSION_CODES.R)
     fun clearClipboardViews() {
         mToolbarHolder?.autofill_suggestions_holder?.removeAllViews()
-        // restore original clipboard toolbar appearance
-        mToolbarHolder?.suggestions_items_holder?.gravity = Gravity.CENTER_HORIZONTAL
-        mToolbarHolder?.clipboard_value?.maxWidth = Integer.MAX_VALUE
+        updateSuggestionsToolbarLayout()
+    }
+
+    private fun updateSuggestionsToolbarLayout() {
+        if (hasInlineViews()) {
+            // make room on suggestion toolbar for inline views
+            mToolbarHolder?.suggestions_items_holder?.gravity = Gravity.NO_GRAVITY
+            mToolbarHolder?.clipboard_value?.maxWidth = resources.getDimensionPixelSize(R.dimen.suggestion_max_width)
+        } else {
+            // restore original clipboard toolbar appearance
+            mToolbarHolder?.suggestions_items_holder?.gravity = Gravity.CENTER_HORIZONTAL
+            mToolbarHolder?.suggestions_holder?.measuredWidth?.also { maxWidth ->
+                mToolbarHolder?.clipboard_value?.maxWidth = maxWidth
```

Comment:
guess we could make this part nicer with using mToolbarHolder?.apply {...} and wrap whole if/else in it too